sesame

```diff
@@ -1,22 +1,48 @@
 -- Assumes sensor on top
 -- Opens Door when ANY player is nearby
-
+ 
 -- wrap sensor, assumes sensor is on top
 sensor = peripheral.wrap("top")
-
--- initalizing players 
+nb = peripheral.wrap("back")
+ 
+-- initalizing players
 players = sensor.getPlayers()
-
+flag = true
```

Comment:
Can we rename this variable to `playMusic` instead of `flag` to make it more indicative of its purpose?